ParisBNF/abb/BNFabb48.xml

```diff
@@ -0,0 +1,187 @@
+                  <collection>d'Abbadie</collection>
+                 <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b100877006#"> BnF Éthiopien d'Abbadie 48</idno>
+                    </msIdentifier>
+               <msContents>
+                  <summary>
+                  </summary>
+
+                     <msItem xml:id="ms_i1">
+                       <locus from="3r" to="6rb"/>
+                        <textLang mainLang="gez"/>
+                        <title type="complete" ref="LIT2630Zenahu"/>
+                     </msItem>
+                              <msItem xml:id="ms_i2">
+                           <locus from="7r" to="157vb"/>
+                                 <title type="complete" ref="LIT3309Qerellos"/>
+                           <textLang mainLang="gez"/>
+                              </msItem>
+                         </msContents>
+               <physDesc>
+                  <objectDesc form="Codex">
+                     <supportDesc>
+                        <support>
+                           <material key="parchment"/>
+                        </support>
+                        <extent>
+                          <measure unit="leaf">159</measure>
+                             <measure unit="leaf" type="blank">2<locus target="#1v #2v"></locus>
+                          </measure>
+                               <dimensions type="outer">
+                              <height unit="mm">235</height>
+                              <width unit="mm">163</width>
+                           </dimensions>
+                        </extent>
+                        <foliation></foliation>
+                        <collation>
+                        </collation>
+                     </supportDesc>
+
+                     <layoutDesc>
+                        <layout columns="2" writtenLines="29"/>
+                     </layoutDesc>
+                  </objectDesc>
+                  <handDesc>
+                   <handNote xml:id="h1" script="Ethiopic">
+                      <seg type="ink">According to <persName ref="PRS3182ContiRo"/>Bad writing, may be 16th century.</seg>
+                        </handNote>
+
+                  </handDesc>
+                  <additions>
+                     <list>
+                        <item xml:id="e1">
+                           <locus target="#2r"/>
+                           <desc type="StampExlibris">Stamp of the Library.</desc>
+                        </item>
+                     </list>
+                    <list>
+                      <item xml:id="a1">
+                        <locus target="#1r"/>
+                         <desc type="Unclear"/>
+                         <p>ዋ፡ ዛፍ፡ በሀ፡ ያቤ፡ አደበጃን፡ ፩ጋ፡ ሸዌ፡ ዕርገቱ፡</p>
+                      </item>
+                       <item xml:id="a2">
+                          <locus target="#2r"/>
+                          <title type="incomlete" ref="LIT3327Melchiz2"/>
```

Comment:
`incomplete`

careful (I do not know why the error is not signalled)

Reply:
corrected, thank you!